src/app/downloads/downloads-chart-container.component.spec.ts

```diff
@@ -53,6 +53,8 @@ describe('DownloadsChartContainerComponent', () => {
         metricsPropertyName, metrics: ep1Downloads}));
       store.dispatch(new ACTIONS.CastlePodcastMetricsSuccessAction({
         id: podcast.id, metricsPropertyName, metrics: podDownloads}));
+      store.dispatch(new ACTIONS.ChartTogglePodcastAction({
```

Comment:
This test was failing for me since the podcast was not charted, so I added this toggle action. It could be that this is a smell indicating an incompatibility with the non-entity implementation.